src/core/public/core_system.test.ts

```diff
@@ -335,14 +306,6 @@ describe('#start()', () => {
 });
 
 describe('#stop()', () => {
-  it('calls legacyPlatform.stop()', () => {
-    const coreSystem = createCoreSystem();
-
-    expect(MockLegacyPlatformService.stop).not.toHaveBeenCalled();
-    coreSystem.stop();
-    expect(MockLegacyPlatformService.stop).toHaveBeenCalled();
-  });
-
```

Comment:
Below on line 388 we are still returning a `legacyTargetDomElement` unnecessarily